mappings/net/minecraft/item/ArmorMaterial.mapping

```diff
@@ -1,11 +1,81 @@
+		COMMENT Returns the {@code name} of this {@link ArmorMaterial}, which is used for locating armor texture files.
+		COMMENT
+		COMMENT <p>
+		COMMENT The return value of this method should be in {@code snake_case}.
+		COMMENT {@link net.minecraft.client.render.entity.feature.ArmorFeatureRenderer} will expect to find an armor
+		COMMENT texture file matching {@code minecraft:textures/models/armor/{material_name}_layer_[1/2].png}
+		COMMENT based off the return result of this method.
+		COMMENT
+		COMMENT <p>
+		COMMENT Example: given a return value of {@code cool_material}, {@code ArmorFeatureRenderer} will require a file
+		COMMENT at {@code minecraft:textures/models/armor/cool_material_layer_1.png} and {@code minecraft:textures/models/armor/cool_material_layer_2.png}.
+		COMMENT
+		COMMENT @return the {@code name} of this armor material in snake_case, used for finding armor textures
 	METHOD method_7695 getRepairIngredient ()Lnet/minecraft/class_1856;
+		COMMENT Returns the {@link Ingredient} used to repair items using this {@link ArmorMaterial}.
+		COMMENT
+		COMMENT <p>
+		COMMENT By default, {@link net.minecraft.item.ArmorItem} will delegate {@link net.minecraft.item.Item#canRepair(ItemStack, ItemStack)}
+		COMMENT back to this method.
+		COMMENT
+		COMMENT @return the {@link Ingredient} required to repair items with this {@link ArmorMaterial}
 	METHOD method_7696 getDurability (Lnet/minecraft/class_1304;)I
+		COMMENT Returns the total amount of durability points an {@link net.minecraft.item.ArmorItem} using this {@link ArmorMaterial} has.
+		COMMENT
+		COMMENT <p>
+		COMMENT The value returned here will set the {@link net.minecraft.item.Item.Settings} max durability option when passed
+		COMMENT into {@link net.minecraft.item.ArmorItem#ArmorItem(net.minecraft.item.ArmorMaterial, EquipmentSlot, net.minecraft.item.Item.Settings)}
+		COMMENT if the value was not already specified.
+		COMMENT
+		COMMENT @return the total durability an {@link net.minecraft.item.ArmorItem} with this {@link ArmorMaterial} has
 		ARG 1 slot
+			COMMENT the {@link EquipmentSlot} containing the {@link net.minecraft.item.Item} with this {@link ArmorMaterial}
 	METHOD method_7697 getProtectionAmount (Lnet/minecraft/class_1304;)I
+		COMMENT Returns the amount of armor protection points offered by an {@link net.minecraft.item.ArmorItem}
+		COMMENT using this {@link ArmorMaterial} while it is worn by a player.
+		COMMENT
+		COMMENT <p>
+		COMMENT The protection value returned here is applied as an {@link net.minecraft.entity.attribute.EntityAttributeModifier}
+		COMMENT to a player wearing the {@link net.minecraft.item.ArmorItem} piece via the {@link net.minecraft.entity.attribute.EntityAttributeModifier.Operation#ADDITION} modifier.
+		COMMENT
+		COMMENT <p>
+		COMMENT For more information on the formula used to calculate dealt damage based off armor resistance,
+		COMMENT visit the <a href="https://minecraft.fandom.com/wiki/Armor">Minecraft Wiki Armor page</a>.
```

Comment:
Should we allow a link to the minecraft wiki? We've removed all links in the past. #2672

Reply:
I would say no, besides the reason gave in #2672, the wiki contains references to mojmap. Makes no sense to allow it